fog/ingest/server/src/server.rs

```diff
@@ -359,6 +359,19 @@ where
         self.get_ingest_summary().mode == IngestControllerMode::Active
     }
 
+    /// Ask if the server is idle.
+    /// This is a convenience wrapper used in tests.
```

Comment:
That sounds good to me- didn't realize you could add this to a method. I went ahead and added it to other Fog Ingest sever methods that are only meant for tests.

Reply:
I think its okay, we have similar conventions in fog ingest to expose some state queries like this so that you don't need to use RPC for absolutely everything.

If we make it `cfg(feature = "under_test")` then you will have to rebuild the entire server to run the test which costs developer time. Meanwhile in the release mode binary this will get stripped out by the linker since it's not called in release mode.

Reply:
Good call. Can confirm that the overseer integration tests fail if you add this attribute to the methods (compiler says it can't find the methods). I'm going to take them out.